cmd/buildah/bud.go

```diff
@@ -43,18 +47,15 @@ var (
 			Usage: "add global flags for the container runtime",
 		},
 		cli.StringFlag{
-			Name:  "format",
-			Usage: "`format` of the built image's manifest and metadata",
+			Name:  "signature-policy",
+			Usage: "`pathname` of signature policy file (not usually used)",
```

Comment:
Suggestion: change to "pathname of signature policy file to override default (not usually used)" - makes it more apparently why it's not usually used

Reply:
Should we just make this hidden?

Reply:
@nalind WDYT?

Reply:
Lets handle signature-policy in a different PR.

Reply:
@rhatdan  👍 to doing the signature policy in a different PR.  Personally I'd vote to leave it as is, but if we decide to change it there are a number of places that need changing.  The hard part is we really don't want to encourage people to mess with this param outside of testing, but I don't think we want to hide it entirely either.

Reply:
I think this is documented in the man page, so a brief summary is the best we can do here.

Reply:
Right I am suggesting that we hide it and remove the description in the man page.  I don't see why we want to support this.  At least  until customers ask for it.

Reply:
The one case that I've needed it so far is for running a Buildah container that is itself running a Docker registry inside of Travis CI.  But then again I've not gotten that up and running completely.  I don't know if there are or are not other likely use cases for an end user.